code/modules/vehicles/armored/armored_weapons.dm

```diff
@@ -0,0 +1,314 @@
+/obj/item/armored_weapon
```

Comment:
You're probably going to call me cringe, but isn't this and mech weapons similar enough that they could just share a parent type to avoid a whole bunch of copy paste?

Reply:
see discord